src/main/java/controller/RateList.java

```diff
@@ -0,0 +1,98 @@
+package controller;
+
+import model.*;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+/**
+ * Created by aleksey.dobrovolsky on 6/9/2019.
+ */
+public class RateList {
+
+    private static ArrayList<Rate> rateList = createRateList();
+    private static DecimalFormat df2 = new DecimalFormat("#.##");
+    private static Properties p = new Properties();
+
+    private static FileReader readProperties() throws FileNotFoundException {
+        return new FileReader(System.getProperty("user.dir") + "/src/main/resources/rateFilterData");
+    }
+
+    public static void run() throws IOException {
+        System.out.println("Initial list of rates:");
+        controller.RateList.printList(rateList);
+
+        System.out.println("\nSorted by Subscription fee rate list:");
+        controller.RateList.printList(controller.RateList.sortBySubscriptionFee(rateList));
+
+        System.out.println("\nFiltered by the specified condition rate list:");
+        controller.RateList.printList(controller.RateList.filter(rateList));
+    }
+
+    public ArrayList<Rate> getRateList() {
+        return rateList;
+    }
+
+    private static ArrayList<Rate> createRateList() {
+        ArrayList<Rate> list = new ArrayList<>();
+        list.add(new Rate4G());
+        list.add(new RateSuper8());
+        list.add(new RateUltra());
+        list.add(new RateInternetMax());
+        list.add(new RateInternetMin());
+        return list;
+
+    }
+
+    public int getClientCount() {
```

Comment:
unused method

Reply:
done - the method was not called previously. Now it's called in RateListRunner